ImageClassification/ILSVRC2012/Googlenet/googlenet_neon.py

```diff
@@ -122,16 +148,16 @@ def aux_branch(bnode):
                  weights=[1, 0., 0.])  # We only want to consider the CE of the main path
 
 assert os.path.exists(args.model_file), 'script requires the trained weights file'
-model.load_params(args.model_file)
-model.initialize(test, cost)
+model.load_params(args.model_file, load_states=False)
+model.initialize(val, cost)
```

Comment:
In general, not specific to this task, 
what is the difference between validation and test? 
Normally we do training + validation. And then we do test with test dataset, right? 
It seems neon has only being using training dataset and validation dataset for testing, not test dataset.